tests/openssl_unsupported/enc/CMakeLists.txt

```diff
@@ -41,21 +41,26 @@ function (add_unsupported_test NAME)
             --config $<CONFIGURATION>
     WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
 
-  # In addition to expecting the compilation to fail, checking whether the log includes
-  # the specific error message, which indicates the unsupported.h header is
-  # correctly included.
-  set_tests_properties(
-    openssl_unsupported_${NAME}
-    PROPERTIES
-      WILL_FAIL
-      TRUE
-      FAIL_REGULAR_EXPRESSION
-      "warning: .* is deprecated: The function|macro may be unsafe inside an enclave"
-  )
+  # The compilation of the baseline case (i.e., not linking against any of unsupported APIs
+  # but simply including the headers) is expected to succeed.
+  if (NOT ${NAME} STREQUAL "baseline")
```

Comment:
Does this work? CMake `STREQUAL` is always case sensitive, isn't it?

Reply:
The `NAME` is converted to the lower case earlier in this function: https://github.com/openenclave/openenclave/blob/30070f4aa116050338a59d740650d16d0c082dbd/tests/openssl_unsupported/enc/CMakeLists.txt#L16